esmvaltool/interface_scripts/interface.ncl

```diff
@@ -34,16 +39,111 @@ begin
   config_user_info@run_dir = config_user_info@run_dir + "/"
   config_user_info@work_dir = config_user_info@work_dir + "/"
 
-  ; Copy some info into legacy names
-  diag_script = diag_script_info@script
-  if isvar("variable_info") then
-    variables = (/variable_info@short_name/)
-    field_types = (/variable_info@field/)
-  end if
+  ; Define current script name as global variable (following Python syntax)
+  _DIAG_SCRIPT = diag_script_info@script
```

Comment:
One little comment: in my understanding leading underscores in python mean that something is "private" and should not be used by the user. This is not the case for these variables, so I suggest removing the underscores.
```suggestion
  DIAG_SCRIPT = diag_script_info@script
```

Reply:
Good point. Let's just use upper-case names for global variables.